Sources/AltSwiftUI/Source/Views/Controls/Menu.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  Menu.swift
+//  AltSwiftUI
+//
+//  Created by Chan, Chengwei on 2021/02/25.
+//  Copyright © 2021 Rakuten Travel. All rights reserved.
+//
+
+import UIKit
+
+@available(iOS 14.0, *)
+/// A view that can be tapped by the user to open a menu.
+public struct Menu: View {
+    public var viewStore = ViewValues()
+    var label: View
+    var viewContent: [View]
+    
+    /// Creates an instance that triggers an `action`.
+    ///
+    /// - Parameters:
+    ///     - - content: A view builder that creates the content of menu options.
+    ///     - label: The visual representation of the menu button
+    public init(@ViewBuilder content: () -> View, @ViewBuilder label: () -> View) {
+        self.label = label()
```

Comment:
since you are not supporting parsing multiple views, I recommend not supporting `@ViewBuilder` in the label property. (Even though Button has a ViewBuilder, it only uses the first view in the group, so I think this is not necessary)